src/components/stack-chart/Canvas.tsx

```diff
@@ -624,6 +625,17 @@ class StackChartCanvasImpl extends React.PureComponent<Props> {
       timelineUnit === 'bytes'
         ? formatBytes(duration)
         : formatMilliseconds(duration);
+    let argumentSummaries = undefined;
+    if (timing.argumentValues) {
+      const argumentValues = timing.argumentValues[stackTimingIndex];
```

Comment:
The `argumentValues` variable is really a `argumentValueIndex` right? Could we maybe rename it to be clearer?